bitnami/rabbitmq-cluster-operator/Chart.yaml

```diff
@@ -37,4 +37,4 @@ maintainers:
 name: rabbitmq-cluster-operator
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq-cluster-operator
-version: 3.7.4
+version: 3.7.5
```

Comment:
Please bump minor version instead since you're adding a new functionality.

Reply:
Thanks. I bumped a minor version.